docs/activities/general/index.html

```diff
@@ -25,7 +25,7 @@ <h3>Task Forces</h3>
 	
 	<h3>Events</h3>
 	<p>
-		The Web of Things (WoT) groups meet during the W3C's annual Technical Plenary week and additional face-to-face meetings are scheduled by the participants. Moreover, the work items are prototyped and tested in WoT IG PlugFests.
+		The Web of Things (WoT) groups meet during the W3C's annual Technical Plenary week and additional face-to-face meetings are scheduled by the participants that happen 2 times a year. Moreover, the work items are prototyped and tested in WoT IG PlugFests that are scheduled for the weekend or week before the face-to-face meetings.
```

Comment:
I am fine with the second part of the change. About the first addition I am not quite sure, 

see [charter](https://www.w3.org/2020/01/wot-wg-charter.html) mentions  "..with no more than four (4) face-to-face"

Reply:
There's no guarantee we will have the plugfest the week before the F2F.  In fact in the one in March we are skipping a week due to IETF.  Could just say "shortly before each F2F" (I think "shortly" is vague enough).